tests/CMakeLists.txt

```diff
@@ -246,7 +250,13 @@ add_executable(
   arangodbtests
   ${ARANGODB_TESTS_SOURCES}
   main.cpp
-)
+  )
+
+add_executable(
```

Comment:
This should be `EXCLUDE_FROM_ALL` as it is covered with the regular tests above for a default build.

https://cmake.org/cmake/help/v3.16/command/add_executable.html